### ISSUE TITLE
Check for theme prefix

### ISSUE THREAD

**Issue by dingo-d** [WPTRT/theme-sniffer#75]
This should be coordinated with the leads from the Theme review team, but if we specify the theme prefix (prefixes) in the `style.css` we could parse it, and then include in the checks using

```php
PHP_CodeSniffer::setConfigData( 'prefixes', implode( ',', $args['theme_prefixes'] ), true );
```

**Comment by grappler** [comment]
I don't think defining the prefixes in the `style.css` header is the right place. It may be better to define it in a `phpcs.xml` file. We would need to be careful that does not override the rules that we have.

**Comment by carolinan** [comment]
But that is also raising the barrier for all theme authors.

**Comment by dingo-d** [comment]
@grappler would that mean that every theme author should include their own `phpcs.xml` in the theme with the specified prefix?

**Comment by grappler** [comment]
@dingo-d It is only an idea not a proposal or anything definitive. It would only be needed for theme authors who do not use the theme slug as the prefix.

**Comment by carolinan** [comment]
What if we required the prefix/namespace to be the themeslug? 
We already have the theme slug,  and a whitelist for frameworks.

**Comment by dingo-d** [comment]
@carolinan I think that we had a meeting about this either last year or at the beginning of this year, and we didn't reach a conclusion.

If we required that theme slug is the same as the theme name but in snake case, that could be used, in theory, in the theme sniffer. But I think that in the latest PR I added an input field where people can add prefix to check against.

**Comment by joyously** [comment]
If you make it a requirement now, you still have to deal with old ones that aren't. So it doesn't really do any good to require it.

**Comment by carolinan** [comment]
Though it would work  if it had been a warning or even notice :)

**Comment by dingo-d** [comment]
Implemented in #97 but needs some checks.

**Comment by dingo-d** [comment]
Fixed in the latest release
